### DISCUSSION TITLE
[help] Can get status to sync for weeks

### DISCUSSION THREAD

**Discussion by maillord** [Chia-Network/chia-blockchain#3348] [Support]
Hi, have been trying to farm chia for weeks but having a constant issue of not syncing.

I've tried opening port 8444 to no avail. 

I'm using Windows 7 (forgive me) on a HP 250 G5 Notebook PC with 8GB RAM, with a 14tb external hard drive for farming. 

I;ve attached the screenshot of my issue, I'm also getting the message 'None of your plots have passed the plot filter yet. But you are currently farming 2.772 TiB'

Help is greatly appreciated on how I can resolve this sync issue.

Cheers people

<img width="709" alt="screenshot1" src="https://user-images.githubusercontent.com/83511915/116790736-d1272600-aaad-11eb-8ac4-fdb50676b23d.png">

**Comment by sargonas** [comment]
With all the spike it network traffic, slow syncing issues appear to be on the rise a bit... have you tried adding any manual nodes? Sometimes adding one of the big major nodes, like `chiadb.net:8444` manually can help "kickstart" your sync. Also, even though you have already opened port 8444 it might not hurt to try to enable UPnP on your router, if it isn't already, just to see if that makes a difference.

**Comment by maillord** [comment]
Thanks mate, I tried opening tag port but it’s says it can be numerical values only... any advice on where I’m going wrong? Cheers again

**Reply by sargonas** [reply]
if you are trying to add it via the GUI, you can scroll down the bottom of the main screen and click "connect to other peers" and input chiadb.net in the first fields and 844 in the other (or another hostname, there are quite a few out there including some regional ones operated by chia.net)

if you are using the CLI, you can type `chia show -a chiadb.net:8444` to add the connection there. Once you add some major node connections, they might eventually "drop off" over time, which is normal as your node randomizes the peers it is connected to, but one of these major nodes may help you kick start the sync and speed things up.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
